addons/website/__manifest__.py

```diff
@@ -204,8 +204,6 @@
             ('include', 'web._assets_helpers'),
             'web_editor/static/src/scss/bootstrap_overridden.scss',
             'web/static/lib/bootstrap/scss/_variables.scss',
-            'website/static/src/scss/website.wysiwyg.scss',
```

Comment:
If `website/static/src/scss/website.wysiwyg.scss` is not removed from `website.assets_wysiwyg` the POC seems to work :)